components/dialog/index.css

```
@@ -177,19 +171,21 @@ governing permissions and limitations under the License.
 
   outline: none; /* Hide focus outline */
 
-  font-size: var(--spectrum-dialog-confirm-description-text-size);
-  font-weight: var(--spectrum-dialog-confirm-description-text-font-weight);
-  line-height: var(--spectrum-dialog-confirm-description-text-line-height);
+  font-size: var(--mod-dialog-confirm-description-text-size,var(--spectrum-dialog-confirm-description-text-size));
+  font-weight: var(--mod-regular-font-weight,var(--spectrum-regular-font-weight));
```

```suggestion
  font-weight: var(--mod-dialog-confirm-description-font-weight, var(--spectrum-regular-font-weight));
```
I saw this mod name in the mods.md list that was referencing a global token. We'll want to give it a name that applies to the dialog.